packages/suite-web/e2e/plugins/bridge.ts

```diff
@@ -1,5 +1,7 @@
-const express = require('express');
-const EventEmitter = require('events');
+// @ts-nocheck
```

Comment:
Would be nicer to refactor it instead of // @ts-nocheck

(if it is easy, but should be)

Reply:
removed no-checks and added some primitive types annotations. Typescript is more like a by-product of this PR, not the main goal, so I  guess it should be enough like this